jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerContextGenerator.java

```diff
@@ -193,4 +206,19 @@ static String joinAsJsonArray(List<String> items) {
 
     return resultString.toString();
   }
+
+  /**
+   * Builds a list of Dockerfile "EXPOSE" items.
+   *
+   * @param exposedPorts the list of ports numbers/ranges to expose
+   * @return a string containing an EXPOSE command for each of the entries
+   */
+  @VisibleForTesting
+  static String makeExposeItems(List<String> exposedPorts) {
```

Comment:
Looks like I messed up the ordering for `joinAsJsonArray` as well, but these static methods should be moved up top.